### DISCUSSION TITLE
Remove all text from UI files and set from python classes

### DISCUSSION THREAD

**Discussion by krzywon** [SasView/sasview#2780] [Ideas]
In #2738, there was a request to modify the UI file instead of programmatically modifying the heading shown in the polydispersity panel. I think we should remove all labels, titles, etc. from all UI files and only set them within the Python classes. Many people who can or will work on SasView will know Python, but they may not be familiar with the Qt developer tools.

**Comment by butlerpd** [comment]
I think the key is that we need to decide on how to do this. I had a similar problem as I recall when I was changing the default value in a widget using the .UI file which sometimes worked and other times seemed to make no difference. The latter behavior ended up being due to it being initialized again in the python file.

That said, I found it easier to find the location in the UI file and change it then to find where it was initialized programmatically? so  not sure I agree with the suggested choice. But again, picking only one method is more important than which one we pick IMO

**Comment by llimeht** [comment]
Is there a desire to have the SasView UI available in languages other than English? If so, do the Qt tools to manage the string catalogues and place the translated strings into the UI have a preferred way of operating? My recollection is that the tooling is `.ui`-centric.
